FDK17プロジェクト/コード/03.サウンド/CSound.cs

```diff
@@ -1676,6 +1740,10 @@ private void tBASSサウンドを作成する_ストリーム生成後の共通
 				throw new Exception( string.Format( "サウンドストリームの周波数取得に失敗しました。(BASS_ChannelGetAttribute)[{0}]", Bass.BASS_ErrorGetCode().ToString() ) );
 			}
 			this.nオリジナルの周波数 = (int) freq;
+
```

Comment:
For safety, moved addition to the list until the very end because the Add operation will cause an event to be raised by the observable collection and then the handler of that event might start working with a CSound object that isn't quite ready. Always raise at the very end.